src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationUtils.java

```diff
@@ -13,8 +13,7 @@ public static boolean hasAppointee(SscsCaseDataWrapper wrapper) {
             && wrapper.getNewSscsCaseData().getAppeal().getAppellant() != null
             && wrapper.getNewSscsCaseData().getAppeal().getAppellant().getAppointee() != null
             && wrapper.getNewSscsCaseData().getAppeal().getAppellant().getAppointee().getName() != null
-            && wrapper.getNewSscsCaseData().getAppeal().getAppellant().getAppointee().getName().getFirstName() != null
-            && wrapper.getNewSscsCaseData().getAppeal().getAppellant().getAppointee().getName().getLastName() != null);
+            && wrapper.getNewSscsCaseData().getAppeal().getAppellant().getAppointee().getName().getFirstName() != null);
```

Comment:
Is it intentional to remove getLastName() check?

Reply:
That's the version of NotificationUtils from Mark's 4307 branch. I'll add it back in.